meetings/2024-10-23.md

```diff
@@ -0,0 +1,154 @@
+* eP: Taking some actions - fits under LWS. Is there interest from the WG to at least see if it is fitting under the work, and making sure we are coordinating that in some way.
+* MDJ: Think we should go with the table that you created, if we need clarifications for some cases - then we should clarify. If there is a contradiction then we should discuss until we agree in the community group. Lets keep working with running code and when we run into a question like that we shouldn't wait on the WG to give us an answer as it will take 2 years. Should keep working on Solid as we have it, and make things interoperable on top of it. Will always be non-interopable things in an incubator and community, we should just document when that is the case and see if it converges. We cannot even really force anybody to converge - so should start with what people build, describe that, and then the WG can look, learn from that, and come up with a spec that hopefully aligns.
+* eP: The table has a column operation that is not part of any existing specs - that is something where ?? and access are not defined. They need to be defined and added to spec
+* MDJ: Yes, but that doesn't contradict what I said. We just need to work with what is in this table and iterate on it.
+
+### DST limbo warning
+
+* eP: some timezones will shift at the end of the month, CG meeting stays pinned to UTC
+
+### Special Topic Meetings
+
+
+#### Summary of CRDT meeting
+
+URL: https://wickie.invisible.college/solid-crdt
+
+* eP: I proposed creation of **CRDT for RDF CG** please support - https://www.w3.org/community/groups/proposed/#crdt4rdf
+* ep: Have discussed, Michiel, myself, Tim, Rahul, Niko from Nextgraph, Michael from Braid were present. Discussed CRDTS specifically w. RDF, and then other things (e.g. text, JSON, etc.). For this community have followed up with people already experience with RDF CRDTS, need to follow up with people with experience with other CRDTS. Also need to prioritise which formats we want to work with beyond RDF. There will be more expertise from other efforts as well, as lots of communities with interest.
+* 
+
+#### Reminder of E2EE meeting on Nov 5th
+
+URL: https://www.w3.org/events/meetings/b277ff65-0aad-425e-bd1d-64758cd4547a/20241022T140000/
+* MDJ: Was supposed to be yesterday, moved to Nov. 5
+
+### Vocab PRs
+
+#### Add solid:proxy to Solid Terms
+
+URL: https://github.com/solid/vocab/pull/94
+* eP: My suggestion for this and the data wallet PR, we shouldn't add tests just because someone has idea for new terms. We should only add terms that are in drafts of the spec. We had some conversations with Sarven about where the requirements should go. He put them in a comment in vocab, I don't see this as appropriate place. We need to follow up over there to move specific requirements over to the approproate draft. If normative should go in a draft not vocab comment.
```

Comment:
```suggestion
* eP: My suggestion for this and the data wallet PR — we shouldn't add tests just because someone has anidea for new terms. We should only add terms that are in drafts of the spec. We had some conversations with Sarven about where the requirements should go. He put them in a comment in vocab; I don't see this as the appropriate place. We need to follow up over there to move specific requirements over to the appropriate draft. If normative, should go in a draft, not a vocab comment.
```